cmd/celestia/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"os"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(fullCmd, lightCmd)
+}
+
+func main() {
```

Comment:
When you run `help`, this comes up: 

```
  completion  generate the autocompletion script for the specified shell
  full
  help        Help about any command
  light
  ```
  
 Any way to remove `completion` option? Not sure what it does. Also it would be nice if `full` and `light` had some sort of description when listed as available commands.